cEP-0005.md

```diff
@@ -96,15 +96,13 @@ from coalib.results.Result import Result
 from coalib.bearlib.aspects import Root
 
 
-class RedundancyBear(LocalBear):
-    # coala will run bears based on this metadata
-    LANGUAGES = {"C", "Python"}
-    # This bear is supposed to be able to fix unused imports...
-    FIX_ASPECTS = {Root.Redundancy.UnusedImport}
-    # ... and detect code clones.
-    DETECT_ASPECTS = {Root.Redundancy.Clone}
+# To define the aspects and languages for a bear using the bear metaclass
+class RedundancyBear(LocalBear, aspects={
+    'detect': [Root.Redundancy.Clone],
+    'fix': [Root.Redundancy.UnusedImport]
+    },languages=['C', 'Python']):
```

Comment:
Still not fully PEP8 compliant :/ Patch from `PEP8Bear`:

```diff
 class RedundancyBear(LocalBear, aspects={
-    'detect': [Root.Redundancy.Clone],
-    'fix': [Root.Redundancy.UnusedImport]
-    },languages=['C', 'Python']):
+        'detect': [Root.Redundancy.Clone],
+        'fix': [Root.Redundancy.UnusedImport]
+        }, languages=['C', 'Python']):
```

In statement expressions creating a new scope below you should always use 8 spaces for continuation... To not make it look like the actual scope below, which is indented by 4

You can also move the closing `aspects={...}` brace to the beginning of the line:

```python
class ...(..., aspects={
        ...
}, languages=['C', 'Python']):
```

But that's just a matter of taste ;)